karrot/invitations/tests/test_prepare_email.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from ..factories import InvitationFactory
+from karrot.invitations.models import Invitation
+from karrot.invitations.emails import prepare_emailinvitation_email
+from karrot.utils.frontend_urls import invite_url
+
+
+class PrepareEmailTestCase(TestCase):
+    def test_prepare_email_called_with_language(self):
+        invitation: Invitation = InvitationFactory(invited_by__language='sw')
+
+        with patch('karrot.invitations.emails.prepare_email') as pe:
```

Comment:
We tend to use the @patch version of it, less indention needed :) An example:

https://github.com/yunity/karrot-backend/blob/cfbcdb905e7ab6a01d5d5c60c21d1f1e8da53abf/karrot/groups/tests/test_receivers.py#L72-L75

Reply:
Also, I'd named it `prepare_email` for clarity.